projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+
+#### Example
+
+In the example below, all the events created through the integration have the same organizer, "Natuurpunt Holsbeek". First you'll need to find the identifier of the organizer `Natuurpunt Holsbeek` in UiTdatabank.
+
+**request**
+
+Perform a `GET` request to the UiTdatabank API including the name of the organizer:
+
+```
+GET /organizers/?q=name.\*:"Natuurpunt Holsbeek"
+```
+
+<!-- theme: warning -->
+
+> For organizers we advice to only search for the `name` and to not include a search on `postalCode` (since this is an optional field for organizers).
+
+**response**
+
+The response now contains the identifier of "Natuurpunt Holsbeek":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116",
+         "@type":"Organizer"
+      }
+   ]
+}
+```
+
+You can now store the identifier of the organizer "Natuurpunt Holsbeek" (`e83edb7f-6f23-47cd-a730-66fd7687f116`) in your application to reuse it as organizer ID in the `POST` request of the event(s) that you want to create.
+
+### Searching existing organizers within the entry form
+
+In this solution you directly integrate with the UiTdatabank Search API in your own entry form.
+
+* ✅ Suitable when the organizers for the events created through your application are unpredictable in advance
+* ❌ Not recommended when the events created through your application all have the same organizer
+* ❌ Not recommended when the events created through your application can have a different organizer, but (1) the amount of organizers is limited & (2) the list of organizers is static and won't change a lot over time
+
+#### Example
+
+In the example below, a user in your entry form types the name "Natuurpunt Holsbeek" in the input field for the organizer. Under the hood an API call to the UiTdatabank Search API is made to check if that organizer already exists:
+
+**request**
+
+```
+GET /organizers/?q=name.\*:"Natuurpunt Holsbeek"
+```
+
+**response**
+
+The response now contains the identifier of "Natuurpunt Holsbeek":
+
+```js
+{
+   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
+   "@type":"PagedCollection",
+   "itemsPerPage":30,
+   "totalItems":1,
+   "member":[
+      {
+         "@id":"https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116",
+         "@type":"Organizer"
+      }
+   ]
+}
+```
+
+If the organizer already exists (like in the example above), it is suggested in the user interface in a human-readable way so that the user can select the organizer and thus choose to reuse the organizer for the creation of his or her event.
+
+Only when no results are found, or the results found do not match the organizer that the user wants to use, a new organizer can be created by the user.
+
+### Provide a mapping when exporting your data to UiTdatabank
```

Comment:
There is no solution like this documented for places. Does that mean that all integrators are required to integrate with Search API in their entry form when the places are unpredictable in advance? I'm not sure a lot will actually do that?

Reply:
I'm very hesitant to document it because I think a mapping for places when exporting is more difficult to execute correctly. This because when all the data is entered already in the database of the integrator, the name of the place is probably a bit different than the one we have in UiTdatabank, for example 
* `Gemeenschapscentrum De Wiek` vs `GC De Wiek` 
or 
* `KMSKA` vs `Koninklijk Museum voor Schone Kunsten Antwerpen` vs `KMSKA - Koninklijk Museum voor Schone Kunsten Antwerpen` vs `Koninklijk Museum voor Schone Kunsten Antwerpen (KMSKA)` vs ..

Because the name will most likely not 100% match the name in UiTdatabank, the integrator can assume that the place does not exist yet and therefore create a new, which will result in a duplicate.

Of course this issue also occurs for organizers to some extent, but there is a double check in the creation of organizers with the `url`. For places we don't have that kind of check.

Reply:
Okay let's leave it out for now and we can always add it later if needed. I suspect that we'll still get some questions about it and we can always fine-tune this then based on the questions/feedback we get.